repository_service_tuf_api/bootstrap.py

```diff
@@ -113,7 +113,7 @@ class Settings(BaseModel):
 
 class BootstrapPayload(BaseModel):
     settings: Settings
-    metadata: Dict[str, TUFMetadata]
```

Comment:
@kairoaraujo we are changing the `BootstrapPayload` expected values.
Shouldn't there be a unit test failure?